service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 +
-[source]
+[source, subs="+quotes"]
 ----
-oc apply -f https://raw.githubusercontent.com/redhat-developer/app-services-guides/main/code-examples/quarkus-kafka-quickstart/.kubernetes/kubernetes.yml
+$ oc login -u __<user>__ -p __<password>__ --server=__<host:port>__
 ----
-+
-. Fetch route to the application by executing
+. Ensure that the current OpenShift project is the one created earlier in this tutorial. For example:
 +
 [source]
 ----
-oc get route
+$ oc project rhoas-quarkus
 ----
-+
-. Open url from **HOST/PORT** section in your default browser
-. Verify that you see Quarkus landing page
-. If the Quarkus application fails to run, review the error log in the terminal and address any problems. Also review the steps in this quick start to ensure that the Quarkus application and Kafka topic are configured correctly.
-. Append `/prices.html` to url to see prices section.
-. You should see `N/A` as price as we still did not connected our application to our Kafka instance
-
-ifdef::qs[]
-.Verification
-* Did the Quarkus example application run without any errors?
-* Can you view `prices.html` page
-endif::[]
-
-[id="proc-binding-kafka_{context}"]
-== Connecting your service with running application
-
-Once application is running we can now connect our service using `rhoas cluster bind` command. This command will let us inject credentials to our application as files. Credentials will be by default injected as volume into our kuberentes deployment. Quarkus Kuberentes Client will detect them and automatically configure our application for us.
-
-Bind command will create following structure in your pod that many frameworks like quarkus can automatically read and autoconfigure your application to connect with the Kafka and other services.
 
+. Use the RHOAS CLI to bind your Kafka instance to your OpenShift project.
++
 [source]
 ----
-/bindings/yourkafka
-├── bootstrapServers
-├── password
-├── provider
-├── saslMechanism
-├── securityProtocol
-├── type
-└── user
+$ rhoas cluster bind
 ----
++
+You are prompted to specify the Kafka instance that you want to connect to OpenShift.
 
-.Procedure
-. Execute
+. Type the name of the Kafka instance that you want to connect to OpenShift. Press *Enter*.
+. Type `y` to confirm that you want to continue. Press *Enter*.
++
+When binding is complete, you should see output like the following:
 +
 [source]
 ----
-rhoas cluster bind
+Binding my-kafka-instance with rhoas-quarkus-kafka app succeeded
 ----
 +
-Command will automatically detect our application and single Kafka service in our namespace and inject connection details to the running application.
-. Select application we want to connect with.
-. Press enter to select it on the list
-. Verify that command finished with **Binding succeeded** message
-. Please go back to your app `prices.html` page
-. You should see prices changing on the webpage
-
-
-ifdef::qs[]
-.Verification
```

Comment:
See above ^^

Reply:
See previous reply. This content does not include Verification sections.